tests/common/libvm.sh

```diff
@@ -17,11 +17,83 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
-# prepares the VM and library for action
+vm_run_sti() {
+    local vmid=$1
+    sti_qemu_lockf=$(pwd)/sti-qemu.lock
+    mkdir inventory-${vmid}
+    cd inventory-${vmid}
+    # Marker to denote this is really a tmpdir
+    # https://fedoraproject.org/wiki/CI/Tests
+    if test -z "${TEST_SUBJECTS:-}"; then
+        cat <<EOF
+
+error: TEST_SUBJECTS must be set; e.g.:
+
+  curl -Lo fedora-atomic-host.qcow2 'https://getfedora.org/atomic_qcow2_latest'
+  export TEST_SUBJECTS=\$(pwd)/fedora-atomic-host.qcow2
+
+If you're doing interactive development, we recommend caching the qcow2
+somewhere persistent.
+EOF
+        exit 1
+    fi
+    for subj in ${TEST_SUBJECTS}; do
```

Comment:
What does it even mean for us if `TEST_SUBJECTS` includes multiple qcow2s? Should we check for that and error out if so?

Reply:
Feels like a bit of a "doctor it hurts when I..." situation?  I started reading [a stackoverflow thing on bash arrays](https://stackoverflow.com/questions/10586153/split-string-into-an-array-in-bash) but then my eyes started glazing over...